jasper/data/locale/fr-FR.po

```diff
@@ -0,0 +1,36 @@
+# Jasper core translations
+# Copyright (C) 2015 Shubhro Saha, Charlie Marsh & Jan Holthuis
+# This file is distributed under the same license as the Jasper package.
+# Jan Holthuis <holthuis.jan@googlemail>, 2015.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: 2.0-dev\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-10-28 01:17+0100\n"
+"PO-Revision-Date: 2015-10-28 01:23+0100\n"
+"Last-Translator: Jan Holthuis <holthuis.jan@googlemail.com>\n"
+"Language-Team: Jasper Project\n"
+"Language: en_US\n"
```

Comment:
Don't forget to edit Metadata (`Language`, `Last-Translator` and `PO-Revision-Date`).